web/modules/custom/customnewblock/customnewblock.info.yml

```diff
@@ -0,0 +1,7 @@
+name: Custom New Block
+description: Creating a module for custom block
+type: module
+core: 8.x
+core_version_requirement: ^8 || ^9
+dependencies:
+  - block
```

Comment:
@anitha-innoraft - Dependencies should specify there package as well i.e. it would be - `drupal:block`

Reply:
Done